integration-tests/src/test/resources/application.conf

```diff
@@ -0,0 +1,16 @@
+# TODO replace it with hosts discovery
+hosts = ["50.18.91.185", "54.241.96.75", "54.153.35.114"]
```

Comment:
IMO it should be taken from the terraform output for the given workspace

Reply:
I think it's better to take it from loadbalancer. In the same way that any user would do it.

Reply:
It depends on the scenario. Most likely we should pass all nodes to the tests but if scenario require a load balancer, we should use load balancer (but keeping in mind all the nodes which are building a test cluster). i.e. you cannot check node joining based on the load balancer only. it will show you nothing

Reply:
Yeah, sometimes you need to query specific endpoint to for example test joining. Querying load balancer only shows the result of one random node from upstream but I guess you may need to check all the nodes.

Reply:
@kpudlik fully agree. The intention was to use LB only for node discovery, with [/cluster/info endpoint](http://lb.testnet.constellationnetwork.io:9000/cluster/info).

Reply:
Resolved, the output now is taken from the terraform.